ui/app/src/state/epics/preview.ts

```diff
@@ -89,7 +90,8 @@ export default [
           getHostToGuestBus().next(setHighlightMode(action.payload));
         }
       }),
-      ignoreElements()
+      filter(([{ payload }, state]) => !payload.editMode && Boolean(state.preview.guest?.path)),
```

Comment:
Maybe we should lock here too?
And then, we would need to do it on the initialization too?

Reply:
this is being called on init